### PR TITLE
Hotfix & improvements

### PR DESCRIPTION
**Pull request by victor-paumier**
- fix nbBuddies history
- show users with deleted university 
- show section with no users (admin view)
- fix admin create university (admin view)

### CODE REVIEW

File: src/Command/FixBDDCommand.php [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    {
+        $this
+            ->setName('buddy:fixBDD')
+            ->setDescription('Fix BDD')
+        ;
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output)
+    {
+        ini_set("memory_limit", "-1");
```
Comment (JeremieSamson) [inline src/Command/FixBDDCommand.php]: Wowowowowowowo what the hell dude

File: src/DataFixtures/UserFixtures.php [hunk 2]
```diff
@@ -93,7 +93,7 @@ public function load(ObjectManager $manager)
             $is->setArrival($now);
             $is->setTermsOfUse(1);
             $is->setUniversity($university);
-            $is->setNbBuddyWanted(rand(1, $is->getSection()->getNbMentors()));
+            $is->setNbBuddiesWanted(rand(1, $is->getSection()->getNbMentors()));
```
Comment (JeremieSamson) [inline src/DataFixtures/UserFixtures.php]: :medal_military:

File: src/Entity/User.php [hunk 3]
```diff
@@ -232,26 +225,12 @@ class User extends BaseUser
...
      *
-     * @ORM\Column(type="integer")
-     */
-    private $nbBuddyWanted = 1;
-
-    /**
-     * @var string
-     *
-     * @ORM\Column(nullable=true)
+     * @ORM\Column(type="integer", options={"default" : 0, "unsigned" : true})
```
Comment (JeremieSamson) [inline src/Entity/User.php]: maybe we should have default 1 here ?

Comment (victor-paumier) [reply src/Entity/User.php]: done

**Pull request merged**